src/components/TimePicker/styled/selectValue.js

```diff
@@ -1,12 +1,21 @@
 import styled from 'styled-components';
-import { COLOR_BRAND, COLOR_WHITE, COLOR_BRAND_ACTIVE } from '../../../styles/colors';
-import { SHADOW_OUTLINE } from '../../../styles/shadows';
-
-const StyledSelectValue = styled.input`
+import getTheme from '../../../styles/helpers/getTheme';
+
+const StyledSelectValue = styled.input.attrs(props => {
+    const theme = getTheme(props);
+    const { getContrastText, brand } = theme.palette;
+    const { main: brandMainColor, dark: brandDarkColor } = brand;
+
+    return {
+        brandMainColor,
+        brandDarkColor,
+        getContrastText,
```

Comment:
Since getContrastText is called several times in the styles, here we should inject the contrast text computed only once like:
```
const brandMainContrastText = getContrastText(props.brandMainColor);
const brandDarkContrastText = getContrastText(props.brandDarkColor);

return {
        brandMainColor,
        brandDarkColor,
        brandMainContrastText,
        brandDarkContrastText,
};
```